tests/operator/operator.go

```diff
@@ -1927,6 +1931,156 @@ spec:
 		)
 	})
 
+	Describe("[level:component]downwardMetrics", func() {
+
+		It("should be able available after update", decorators.Upgrade, func() {
+			if !libstorage.HasCDI() {
+				Skip("Skip update test when CDI is not present")
+			}
```

Comment:
Is this Skip required? Shouldn't the `decorators.Upgrade` label enough? If not, I would move this in a `BeforeEach`